Dixit_Logic/Interfaces/IDixitGame.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dixit_Logic.Interfaces
+{
+    /// <summary>
+    /// This is a game's round interface. It cointain what need to a round of dixit game.
+    /// </summary>
+    interface IDixitGame
+    {
+        /// <summary>
+        /// It contain all players who playe in the acutal game. 
+        /// </summary>
+        IPlayer[] Players
```

Comment:
Please use List here, and other places as well. Consider using IReadonlyList where applicable.